docs/userguide/containers/dockerrepos.md

```diff
@@ -42,6 +42,7 @@ You can now commit and push your own images up to your repos on Docker Hub.
 > **Note:**
 > Your authentication credentials will be stored in the `~/.docker/config.json`
 > authentication file in your home directory.
+> For Windows users, this Docker file is stored in `$HOME/.docker/config.json`
```

Comment:
@lirantal If you want to use the gist provided and update your PR, you can submit this communal decision as your contribution. If not, let us know and we will carry.  Thanks for raising this issue.